osu.Game/Overlays/Settings/Sections/AudioSection.cs

```diff
@@ -22,13 +23,18 @@ public class AudioSection : SettingsSection
 
         public AudioSection()
         {
-            Children = new Drawable[]
+            List<Drawable> children = new List<Drawable>();
+            children.Add(new AudioDevicesSettings());
+            children.Add(new VolumeSettings());
+            children.Add(new OffsetSettings());
+            if (!(RuntimeInformation.IsOSPlatform(OSPlatform.Windows) || RuntimeInformation.IsOSPlatform(OSPlatform.OSX)))
```

Comment:
* use `osu.Framework.RuntimeInfo` instead.
* this should probably be `RuntimeInfo.OS == RuntimeInfo.Platform.Linux`. i don't believe this should show on mobile platforms, either.

Reply:
not wanted on macOS either. this is linux specific.